torch/fx/passes/graph_drawer.py

```diff
@@ -215,6 +215,8 @@ def _get_str_for_args_kwargs(arg):
                 if len(arg_strs_list) == 0:
                     return ""
                 arg_strs = prefix + r",\n".join(arg_strs_list) + suffix
+                if len(arg_strs_list) == 1:
```

Comment:
why do we need this ?

Reply:
It's just formatting to make the svg graph more compact when there is exactly one field to format.